### ISSUE TITLE
When indexing, ignore too large files

### ISSUE THREAD

**Issue by ChristopheBordieu** [oracle/opengrok#3478]
When indexing, it is possible to skip files of certain types (or even with some precise paths I think) with -i option.
I would need a way to skip any file bigger than a size I could define myself if your default is not convenient for me.

**Comment by vladak** [comment]
dup of #534 ?

**Comment by ChristopheBordieu** [comment]
Yes... And I didn't remember I had already opened an issue for this enhancement (almost 3 years ago): #1646
